homeassistant/components/notify/yessssms.py

```diff
@@ -38,14 +39,35 @@ def __init__(self, username, password, recipient):
         from YesssSMS import YesssSMS
         self.yesss = YesssSMS(username, password)
         self._recipient = recipient
+        # pylint: disable=protected-access
+        _LOGGER.debug(
+            "initialized; library version: %s", self.yesss._version)
 
     def send_message(self, message="", **kwargs):
         """Send a SMS message via Yesss.at's website."""
+        if self.yesss.account_is_suspended():
+            # only retry to login after HASS was restarted with (hopefully)
+            # new login data.
+            _LOGGER.critical(
```

Comment:
This should be an error log level. Critical is reserved for bugs which basically mean hass is crashing, such as out of memory.